plugins/astroturf/README.md

```diff
@@ -0,0 +1,50 @@
+# @poi/plugin-astroturf
+
+Use [astroturf](https://github.com/4Catalyzer/astroturf) to write CSS in JS with zero-runtime.
+
+## Install
+
+```bash
+yarn add astroturf @poi/plugin-astroturf --dev
+```
+
+## How to use
+
+In your `poi.config.js`:
+
+```js
+module.exports = {
+  plugins: [
+    {
+      resolve: '@poi/plugin-astroturf',
+      options: {}
+    }
+  ]
+}
+```
+
+Then write css-in-js:
+
+```jsx
+import { css } from 'astroturf'
+
+const style = css`
+  .title {
+    font-size: 2em;
+  }
+`
+export const App = () => {
+  return <div className={style.title}>Hello world</div>
+}
+```
+
+Check out [astroturf](https://github.com/4Catalyzer/astroturf/) for more usages.
+
+## Options
+
+- `loaderOptions`: `astroturf/loader` [options](https://github.com/4Catalyzer/astroturf/tree/1741c3e702049f6e75483cd000b439d42e57ef2d#options)
+  - `loaderOptions.extension`: Default `.module.css`. You can also change it to `.module.scss` etc to use [CSS pre-processors](https://poi.js.org/guide/pre-processing-css.html), but make sure that it ends with `.module.xxx` to get proper CSS modules support.
```

Comment:
@egoist If you write code like this:

```js
// poi.config.js

module.exports = {
  plugins: [
    {
      resolve: '@poi/plugin-astroturf',
      options: {
        loaderOptions: {
          extension: '.scss'
        }
      }
    }
  ]
}
```

```jsx
import { css } from 'astroturf'

css`
  .title {
    font-size: 2em;
  }
`
export const App = () => {
  return <div className="title">Hello world</div>
}
```

It can also work

Reply:
<img width="1082" alt="屏幕快照 2019-04-03 下午4 11 31" src="https://user-images.githubusercontent.com/8784712/55463256-312b1400-562b-11e9-8225-c9ccdc04913d.png">

doesn't work for me, isn't css modules a requirement?

Reply:
oh I get what you mean

Reply:
but their default css-loader always has css modules enabled: https://github.com/4Catalyzer/astroturf/blob/5664f9c080db645f0579153cd7e96522fe5515b4/src/css-loader.js#L14

Reply:
@egoist Yes, it's weird, their css-loader even not compatible with css-loader 2.x...🤦‍♂️

Anyway, non-css-module works very well in my project 🤷‍♂️. And they also didn't say that must use their css-loader, so I think(or maybe I hope🤣) this will always work.